recipes/qpdf/all/conanfile.py

```diff
@@ -0,0 +1,134 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.51.3"
+
+class PackageConan(ConanFile):
+    name = "qpdf"
+    description = "QPDF is a command-line tool and C++ library that performs content-preserving transformations on PDF files."
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/qpdf/qpdf"
+    topics = ("pdf")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_crypto": ["native", "openssl", "gnutls"],
+        "with_jpeg": ["libjpeg", "libjpeg-turbo", "mozjpeg"],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_crypto": "native",
+        "with_jpeg": "libjpeg",
+    }
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 14
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "5",
+            "Visual Studio": "19.10",
```

Comment:
I think it would be beneficial to update the cmake package template accordingly, since i took it from there and just updated the version which lists C++14 support, as demanded from qpdf.

I am still quite confused what to do here exactly. Should i add another entry for msvc, or should i replace the "Visual Studio" entry? I don't get the desired usage from just the function documentation.

Reply:
Wow! I didn't know those package templates were there. Those are incredibly helpful! I'll see if I can do a PR to update that template later / get feedback on the best way to do that. I use `check_min_vs()` in the `boost-ext-ut` recipe: https://github.com/conan-io/conan-center-index/blob/master/recipes/boost-ext-ut/all/conanfile.py#L59-L68